_posts/2024-02-17-survey-https-infra.md

```diff
@@ -0,0 +1,195 @@
+
+## Donations
+
+As you might know, the OpenTTD infrastructure is fully paid by donations.
+And for years now, we didn't need to do a fundraiser, as you all so gracefully donated enough money in a year to pay the bills.
+What helped, especially in the earlier years, that we were often sponsored by companies with infrastructure related sponsorships.
+
+These days, we enjoy fewer sponsorships.
+This is not always a bad thing; if you are a paying customer, companies are more likely to help you out.
+Especially with infrastructure, this hasn't always been the case during sponsorships.
+
+That said, two companies do deserve a shout-out, as they do sponsor us still, and we greatly appreciate them.
+
+[Pulimi](https://pulumi.com/), to help manage our infrastructure.
+[1Password](https://1password.com/), to ensure we can all store OpenTTD's related passwords safely.
+
+We told them by email, but also here: thank you so much for your on-going sponsorship; it is truly appreciated.
+
+As for the rest of our infrastructure, we are on our own, and the reserve we had is slowly running dry.
+This means it is not unlikely we will need once again a fundraiser sooner or later.
+When that moment happens, we will sure to let you know.
+But till then, I would like to thank you all for your donations; we really do appreciate it, and it keeps the (virtual) lights on.
+
+## Future ahead
+
+As for the future, there is a lot of interest for Cloud Saves: something we are still looking into.
+With Cloud Saves you would be able to save your savegame in the cloud, and continue with it from where ever you are.
+And as with all services OpenTTD offer, it will not be only for Steam users, but it will be available for everyone, regardless of what platform you use.
+
+The main trouble here is to figure out what the costs per player will be, and making sure we can actually pay for that year-to-year.
+As you can read above, although we are financially healthy, we have to ensure we stay that way.
+Which means it will mostly depend whether the changes in 14.0 help in reducing the BaNaNaS bandwidth bill, before we start extending our services to include new, not-cost-free, services.
+
+The main question that is hard to answer: how big will the average savegame be.
+Do most players play small maps? Or very large maps?
+This will be answered by the survey results, and this will help creating a good estimate.
+
+To give you a rough estimate, the costs for Cloud Saves will be roughly 0.02 dollar per GB per month stored.
+If we can assume an average savegame is ~10MB, and we allow storing 10 saves per player, that would mean the average player will cost us 0.002 dollar per month.
+Or 0.02 dollar per year per player.
+On its own not a lot, but if you remember earlier where we talked that we have ~1000 active player per day ... it could add up to a lot of money really fast.
+So we need better metrics to know how many players to expect.
+And whether ~10MB is fair.
```

Comment:
Tangentially related to this post... won't you need to add the last save's file size or something similar in the survey to know the average/median save size?

Reply:
Hmm, size of the current savegame if saved in that session, that would be kinda useful! If I find time before 14.0 I might add it (or if you want to, go for it!)